deploy/templates/nstemplatetiers/test/cluster.yaml

```diff
@@ -3,25 +3,6 @@ kind: Template
 metadata:
   name: test-cluster-resources
 objects:
-- apiVersion: quota.openshift.io/v1
```

Comment:
Moved resource quotas to the namespace, so this seems unnecessary, but I will restore and set to cpu/memory that accomodates the resourcequota from the namespace if you prefer